gamemodes/terrortown/gamemode/shared/sh_voice.lua

```diff
@@ -0,0 +1,14 @@
+---
+-- @section voice_manager
+
+---
+-- Whether or not the @{Player} use the voice chat. Hint: Has to be registered
```

Comment:
`Whether or not the @{Player} use the voice chat.` -> `Whether or not the @{Player} **can** use the voice chat.`

`Hint: Has to be registered` should be more specific, what do you mean? Or remove it if it is not that important.

Reply:
The hook has to be registered on both client and server to make it work, see the next line

Reply:
That isn't the code style or doc style we should go. Give a look into `@warning` or `@realm`

Reply:
There is already `@realm shared` i just wanted to mention that the hook has to be registered on both client and server to function properly. I guess I could add a `@warning` though

Reply:
There is already `@realm shared` i just wanted to mention that the hook has to be registered on both client and server to function properly. I guess I could add a `@warning` though

Reply:
That's a normal behaviour if using predicted hooks with different values on client and server.
Same with sprinting